Gordon360/Services/RecIM/SeriesService.cs

```diff
@@ -10,6 +10,7 @@
 using System.Threading.Tasks;
 using Gordon360.Extensions.System;
 using Gordon360.Static.Names;
+using Microsoft.Extensions.FileSystemGlobbing;
```

Comment:
What is this used for?

Reply:
I'm not going to lie, I got no clue what those words even mean. (jk I kind of do, but I don't know how that pertains to the seriesService)

Probably just another thing that got added from typing and pressing tab